app/(dashboard)/[category]/list/unreadMessagesBadge.tsx

```
@@ -0,0 +1,21 @@
+}
+
+export const UnreadMessagesBadge = ({ count, className }: UnreadMessagesBadgeProps) => {
+  if (count <= 0) return null;
+
+  const displayText = count > 10 ? "10+ unread messages" : `${count} unread message${count === 1 ? "" : "s"}`;
+
+  return (
+    <Badge 
+      variant="bright" 
+      className={`text-xs px-3 py-1 rounded-full whitespace-nowrap ${className}`}
+    >
```

ESLint

[error] Replace `⏎······variant="bright"·⏎······className={`text-xs·px-3·py-1·rounded-full·whitespace-nowrap·${className}`}⏎····` with `variant="bright"·className={`text-xs·px-3·py-1·rounded-full·whitespace-nowrap·${className}`}`

(prettier/prettier)